src/azure-cli-core/azure/cli/core/auth/identity.py

```diff
@@ -159,7 +159,7 @@ def _service_principal_store(self):
     def login_with_auth_code(self, scopes, **kwargs):
         # Emit a warning to inform that a browser is opened.
         # Only show the path part of the URL and hide the query string.
-        logger.warning("The default web browser has been opened at %s. Please continue the login in the web browser. "
+        logger.warning("A web browser has been opened at %s. Please continue the login in the web browser. "
```

Comment:
Good point.

By the way, in an upcoming feature branch, this behavior will likely change again. :-)  But that is a topic for future. Right now, I think your PR here makes sense. Approving.